tools/generate_requirements.py

```diff
@@ -26,6 +30,32 @@ def generate_requirement_file(name: str, req_list: list[str]) -> None:
     req_fname.write_text("\n".join(header + req_list) + "\n")
 
 
+def generate_environment_yml(req_sections: dict[str, list[str]]) -> None:
+    # Some packages in conda-forge have different names than they do on pypi
+    rename_idx = {
+        'build': 'python-build',
+        'kaleido': 'python-kaleido',
+        'sphinx_design': 'sphinx-design',
+    }
```

Comment:
For cross-reference:

- conda-pypi: https://github.com/conda-incubator/conda-pypi/blob/5f985413b6b69d5c1ddf89840b95318a300b9634/conda_pypi/dependencies/__init__.py#L169-L203
- conda/grayskull#564